src/section.cpp

```diff
@@ -55,6 +55,11 @@ range<const floatType> Section::perimeters() const {
     return get<Property::Perimeter>();
 }
 
+bool Section::hasSameShape(const Section& other) const noexcept {
```

Comment:
`hasSameShape` is a bit arbitrary (it usually refers to dimensions), isn't it? Wouldn't a name such as `hasSameData` or `hasSameValues` be a little more informative?

Reply:
`Shape` I think is closest to what I mean; `Data`/`Value` is too generic; the diameters, etc, all define the shape of the section, but the child count doesn, imo.

Reply:
`hasSameGeometry` maybe?